src/libs/TransactionUtils.js

```diff
@@ -264,7 +264,7 @@ function getLinkedTransaction(reportAction = {}) {
 }
 
 function getAllReportTransactions(reportID) {
-    return _.filter(allTransactions, (transaction) => transaction.reportID === reportID);
+    return _.filter(allTransactions, (transaction) => `${transaction.reportID}` === `${reportID}`);
```

Comment:
@jeet-dhandha why is this change necessary?

Reply:
Transaction's `reportID` was set as number and due to which the filter was not working. As passed id was a string type.

Reply:
where do we use `getAllReportTransactions`?

Reply:
Let's fix the root cause and make sure reportID on the transaction is set as a string. All reportIDs should be strings.

Reply:
where do we use `getAllReportTransactions`?

Reply:
This might need to be handled in back-end. while creating the transactions as on front-end i checked the `reportID` its in string.